### DISCUSSION TITLE
Change app name in browser TAB

### DISCUSSION THREAD

**Discussion by Nikola7i** [serenity-is/Serenity#5548] [Help]
Hi,

How to change name : 

![image](https://user-images.githubusercontent.com/16926063/109383355-53e10880-78e6-11eb-9f88-fdd855529b6d.png)

Thanks.

**Comment by arunkumar12712** [comment]
try this  ViewData["Title"] = "Dashboard";

**Reply by Nikola7i** [reply]
Thank's .. It way in layout...   👍
